src/main/sensors/gyro.c

```diff
@@ -1310,3 +1312,30 @@ void dynLpfGyroUpdate(float throttle)
     }
 }
 #endif
+
+#ifdef USE_YAW_SPIN_RECOVERY
+void setYawSpinRecovery(int maxYawRate)
```

Comment:
Should this be `initYawSpinRecovery()`, as it is not actually a setter?

Reply:
Well it’s not only called at “init”. It can change the settings at runtime if the rates change. Like switching rateprofiles in flight.

Reply:
That applies to `initRcProcessing()`, where this is called from. In Betaflight 'init' is used to designate functions that need to be called when something needs to be reinitialised, as well as functions that are only called for the initial initialisation.

Reply:
I don't have that strong an attachment and think it could go either way. I pushed an update to change it to `initYawSpinRecovery`.